components/toast_wrapper/toast_wrapper.tsx

```diff
@@ -383,10 +400,10 @@ class ToastWrapper extends React.PureComponent {
             showActions: !atLatestPost || (atLatestPost && !atBottom),
         };
 
-        if (showUnreadToast && unreadCount > 0) {
+        if (showUnreadToast && unreadCount as number > 0) {
```

Comment:
If you check `unreadCount` before, you don't need to cast as number, and it is safer. 
```suggestion
        if (showUnreadToast && unreadCount && unreadCount > 0) {
```